pkgs/tools/misc/hostmux/default.nix

```
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, installShellFiles
+, openssh
+, tmux
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "hostmux";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "hukl";
+    repo = "hostmux";
+    rev = finalAttrs.version;
+    hash = "sha256-odN7QFsU3MsWW8VabVjZH+8+AUFOUio8eF9ORv9iPEA=";
+  };
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  buildInputs = [
+    openssh
+    tmux
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 hostmux $out/bin/hostmux
+    installManPage man/hostmux.1
+    installShellCompletion --zsh zsh-completion/_hostmux
+
+    runHook postInstall
+  '';
+
+  postPatch = ''
+    substituteInPlace hostmux \
+      --replace "SSH_CMD=ssh" "SSH_CMD=${openssh}/bin/ssh" \
+      --replace "tmux -2" "${tmux}/bin/tmux -2" \
+      --replace "tmux split-window" "${tmux}/bin/tmux split-window" \
+      --replace "tmux select-layout" "${tmux}/bin/tmux select-layout" \
+      --replace "tmux show-window-option" "${tmux}/bin/tmux show-window-option" \
+      --replace "tmux set-window-option" "${tmux}/bin/tmux set-window-option" \
+      --replace "tmux send-keys" "${tmux}/bin/tmux send-keys"
```

```suggestion
      --replace "tmux " "${tmux}/bin/tmux "
```
Note the space after tmux.
I believe that should suffice.

---

no this would also replace "USAGE= hostmux -h" but i found a smaller version with "tmux s" and "tmux -2"

---

Thanks I didn't spot this!